Add setting for mailing address.

peter-fogg opened the pull request:
Necessary for [ECOM-3318](https://openedx.atlassian.net/browse/ECOM-3318) and https://github.com/edx/edx-platform/pull/11102.

@edx/ecommerce

rlucioni commented on playbooks/roles/edxapp/defaults/main.yml:
```diff
@@ -831,6 +831,7 @@ generic_env_config:  &edxapp_generic_env
   SESSION_SAVE_EVERY_REQUEST: "{{ EDXAPP_SESSION_SAVE_EVERY_REQUEST }}"
   SOCIAL_SHARING_SETTINGS: "{{ EDXAPP_SOCIAL_SHARING_SETTINGS }}"
   SESSION_COOKIE_SECURE: "{{ EDXAPP_SESSION_COOKIE_SECURE }}"
+  CONTACT_MAILING_ADDRESS: "{{ EDXAPP_CONTACT_MAILING_ADDRESS }}"
```
This looks like an LMS specific setting. Since we don't appear to need it in Studio settings, I'd put it in `lms_env_config`.

feanil commented on playbooks/roles/edxapp/defaults/main.yml:
```diff
@@ -248,7 +248,7 @@ EDXAPP_FACEBOOK_APP_ID: "FACEBOOK_APP_ID"
 EDXAPP_FACEBOOK_API_VERSION: "v2.1"
 
-
+EDXAPP_CONTACT_MAILING_ADDRESS: '141 Portland St. Cambridge, MA 02139'
```
This is an edx specific default, are we ok with other deployments having this as their mailing address if they do nothing?

peter-fogg replied:
@feanil No, only our deployment should have this set. Does that mean I should set this to `''` here, and file a ticket to update configuration-secure?

clintonb replied:
I'd set the default to something obviously wrong (e.g. SET-ME-PLEASE).

feanil replied:
+1 to what @clintonb said.  Set it to something obviously wrong and then file a ticket for setting it in secure.  Make sure to mention which deployments you need it to be set it.

clintonb commented:
:+1:

peter-fogg commented:
@clintonb @rlucioni @feanil I've moved the setting to `lms_env_config`, updated its value to `'SET-ME-PLEASE'`, and filed [DEVOPS-3450](https://openedx.atlassian.net/browse/DEVOPS-3450).

rlucioni commented:
:+1: pending the outcome of conversation on https://github.com/edx/edx-platform/pull/11102.
